src/util/getUrl.ts

```diff
@@ -1,13 +1,11 @@
-
 import Logger from "bunyan";
 import url from "url";
+import envVars from "../config/env";
 
 export const getGitHubConfigurationUrl = (
-	githubHost,
-	jwt,
 	jiraHost
 ): string =>
-	`https://${githubHost}/github/configuration?jwt=${jwt}&xdm_e=${jiraHost}`;
+	`https://${jiraHost}/plugins/servlet/ac/com.github.integration.${envVars.INSTANCE_NAME}/github-post-install-page`;
```

Comment:
I feel that this is the crux, but I don't understand why. Is this where the `jiraHost` was lost? So now, we're redirecting to the Jira page, which "inject" the `jiraHost` into the iFrame automatically?

Reply:
Not at all.  This is just a function that returns the URL of where to go after we hit `/github/setup` when we install the app from the github side.  I change this because there's no way to get the JWT token from the `/github/setup` page, so instead of redirecting to a url that won't work, I'm going to redirect them to the jiraHost plugin directly since that injects the jwt token into the iframe.

This isn't related to 3rd party cookies, this is just something I noticed along the way that's adjacent.  Plus, we don't store the jwt token in the session anymore, so this was broken.

Reply:
Do we have jiraHost here if we installed the app from GitHub?

Reply:
We may, but more often than not, no.  Which is why there's logic before it to either automatically redirect if `jiraHost` is known or ask for the jira domain if it's not.